pkg/scheduler/generic_scheduler.go

```diff
@@ -141,7 +130,7 @@ func (g *genericScheduler) Schedule(ctx context.Context, extenders []framework.E
 
 // selectHost takes a prioritized list of nodes and then picks one
 // in a reservoir sampling manner from the nodes that had the highest score.
-func (g *genericScheduler) selectHost(nodeScoreList framework.NodeScoreList) (string, error) {
+func (sched *Scheduler) selectHost(nodeScoreList framework.NodeScoreList) (string, error) {
```

Comment:
It seems like selectHost is stateless, should we refactor this to a general function like `prioritizeNodes`
```golang
func selectHost() {
}
```
Then we can remove the scheduler initialization from tests.

Reply:
Thanks for the review!

I have changed `selectHost` stateless